### ISSUE TITLE
Question: Are Texture set's global?

### ISSUE THREAD

**Issue by oyster-catcher** [shadowmage45/TexturesUnlimited#63]
The mod is great!
One issue is confusing me, are textures sets global across separate mods?
I asked because I used the ProbesPlus mod which worked with texture switching.
However, when adding textures to my own mod I was able to stop the texture switching working for ProbesPlus, note that I did change "Coatl Aerospace" in the Compatibility/TexturesUnlimited.cfg file.

In your documentation you say
```And its texture-sets refer to root-node texture sets, constructed such as:```
does root-node textures sets mean they are global?

**Comment by shadowmage45** [comment]
>does root-node textures sets mean they are global?  

Yes, root-node texture-sets are 'global', and once defined, can be referenced by any module that uses them.  There is a per-mod-texture-set functionality coming in the near future that should allow for more segregation and prevent name collisions.

> note that I did change "Coatl Aerospace" in the Compatibility/TexturesUnlimited.cfg file.

Well, when you change stuff... expect that things break :)

**Comment by oyster-catcher** [comment]
Ok brilliant. Glad to know for sure. Thanks for the reply.
